content/fast/minify-css/index.md

```diff
@@ -55,104 +55,115 @@ In practice, you would like to introduce this technique as one of the steps of y
 ```
 
-Inside **module.exports**, insert:
+Then, pass an instance of the plugin to the **plugins** array:
 ```
-optimization: {
-	minimizer: [
-		new OptimizeCSSAssetsPlugin({})
-	]
-}
+  plugins: [
+    new HtmlWebpackPlugin({template: "./src/index.html"}),
+    new MiniCssExtractPlugin({filename: "[name].css"}),
+    new FixStyleOnlyEntriesPlugin(),
+    new OptimizeCSSAssetsPlugin({})
+  ]
 ```
 After making the changes a rebuild of the project will be triggered.
 This is how the resulting **webpack.config.js** will look like:
 
 <div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
   <iframe
     allow="geolocation; microphone; camera; midi; vr; encrypted-media"
-    src="https://glitch.com/embed/#!/embed/minify-css-optimized?path=webpack.config.js&previewSize=0"
-    alt="minify-css-optimized on Glitch"
+    src="https://glitch.com/embed/#!/embed/fav-kitties-animated-min?path=webpack.config.js&previewSize=0"
+    alt="fav-kitties-animated-min on Glitch"
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
 
-1. Go back to the console, and run `cd public`.
-1. Run `vi main.css`, to inspect the content of the CSS.
+Next, you'll check the result of this optimization with performance tools.
 
-You'll find a more compact version of the CSS, after minification has been applied.
+## Monitor
 
-<div class="aside note">
-Note: This demo is based on Webpack 4. It's likely that Webpack 5 will come with a CSS minimizer built-in. If that’s the case you won’t need to configure a plugin as done in the previous step.
-</div>
+First, run Lighthouse on the [optimized site](https://fav-kitties-animated-min.glitch.me/). 
 
-## Monitor
+The report doesn't show "Minify CSS" as "Opportunity" anymore, and has now moved to "Passed Audits" section:
+
+<img class="screenshot" width="700px" height="150px" src="./lighthouse-optimized.png" alt="Lighthouse Passed Audits for optimized page.">
+
+To inspect the size and content of the files:
 
-1. Open the resulting [optimized page](https://minify-css-optimized.glitch.me/) in Chrome, and open DevTools.
+1. Open the resulting [optimized page](https://fav-kitties-animated-min.glitch.me/) in Chrome, and open DevTools.
 1. Click in the **Network** panel and filter for “CSS”.
 1. Make sure "Disable Cache" is checked and reload the page.
 
-<img class="screenshot" width="510px" height="96px" src="./cdt-css-optimized.png" alt="DevTools CSS unoptimized response">
+<img class="screenshot" width="700px" height="120px" src="./cdt-css-optimized.png" alt="DevTools CSS unoptimized response">
 
-Once minified, [the new CSS file](https://minify-css-optimized.glitch.me/main.css) is now **~24% smaller** than its initial version.
+You can inspect these files, and see that the new versions don't contain any whitespaces. Both files are much smaller, in particular, the [animate.css](http://fav-kitties-animated-min.glitch.me/animate.css) has been reduced in **~26%**, saving **~20KB**!
```

Comment:
Suggest breaking the second sentence in two: "much smaller. In particular, ..."
whitespaces => whitespace
reduced in => reduced by